tools/devops/automation/scripts/update-locproject.ps1

```diff
@@ -15,10 +15,14 @@ $projectObject = Get-Content $LocProjectPath | ConvertFrom-Json
 $jsonFiles | ForEach-Object {
     $sourceFile = $_.FullName
     $outputPath = "$($_.DirectoryName + "\")"
+    $fullNameString = Convert-Path -Path $_.FullName
+    $maciosPath = $fullNameString -split "xamarin-macios", 2
+    $lclFile = "$($LocalizeDirectory + $maciosPath[1] + ".lcl")"
```

Comment:
Lets concatenate files properly. What happens if LocalizeDirectory does not end with a path separator? In this case it will return the wrong path.
Better do:
```pwsh
Join-Path -Path $LocalizeDirectory -ChildPath "$($maciosPath[1]).lcl"
```
Read the docs here: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-7.2

There is also a nice way on how I interpolated the string using "$()" in the example.